app/controllers/concerns/idv/step_indicator_concern.rb

```diff
@@ -31,14 +29,18 @@ def in_person_proofing?
     def gpo_address_verification?
       # Proofing component values are (currently) never reset between proofing attempts, hence why
       # this refers to the session address verification mechanism and not the proofing component.
-      !!current_user.pending_profile || idv_session.address_verification_mechanism == 'gpo'
+      return true if current_user&.pending_profile
+
+      return idv_session&.address_verification_mechanism == 'gpo' if defined?(idv_session)
     end
 
     def proofing_components_as_hash
       # A proofing component record exists as a zero-or-one-to-one relation with a user, and values
       # are set during identity verification. These values are recorded to the profile at creation,
       # including for a pending profile.
       @proofing_components_as_hash ||= begin
+        return {} if !current_user
```

Comment:
Recommend extracting this into a method.

Reply:
Done in [2cbd4bc](https://github.com/18F/identity-idp/pull/8480/commits/2cbd4bcc6762c5a7b36ef5994dcebe4534533715)